docs/source/reference/api-only.md

```diff
@@ -0,0 +1,124 @@
+(api-only)=
+
+# Deploying JupyterHub in "API only mode"
+
+JupyterHub is a service for deploying and managing Jupyter servers for users.
+This functionality is exposed _primarily_ via a [REST API](rest).
+JupyterHub also ships with a _basic_ web UI built on that API,
+enabling users to click a button to quickly start and stop their servers,
+and admins to perform some basic user and server management tasks.
+
+The REST API has always provided functionality not available in the UI,
+and we try to avoid having any functionality only available in the UI and not the API.
+With JupyterHub 2.0, this should be fully the case -
+no UI pages should rely on information not available via the REST API.
+Previously, some admin UI functionality could only be achieved via admin pages,
+such as paginated requests.
```

Comment:
With JupyterHub 2.0, web pages will no longer provide functionality that is unavailable by API.